### ISSUE TITLE
[Bug] Dashboard filters sends filter value as string ( regression of #3816 ? )

### ISSUE THREAD

**Issue by valtlfelipe** [metabase/metabase#7115]
- Your browser and the version: 
Chrome 65.0.3325.146
- Your operating system:
macOS 10.12.6 
- Your databases:
PostgreSQL 9.5.10
- Metabase version:
Latest 0.28.1
- Metabase hosting environment:
Elastic Beanstalk
- Metabase internal database:
PostgreSQL 9.6.6

- *Repeatable steps to reproduce the issue*

1. Create a question and add it to a dashboard
2. Add dashboard filter that filters by a numeric column (doesn't matter if it's a PK or category)
3. Enter filter value (default or user entry - doesn't matter)

Expected: updated, filtered values
Actual: There was a problem displaying this chart.
Error: `ERROR: operator does not exist: bigint = character varying↵ Hint: No operator matches the given name and argument type(s). You might need to add explicit type casts.`

Payload:
```json
{"parameters":[{"type":"id","target":["dimension",["fk->",42,209]],"value":"532"}]}
```

Stack trace:
![](https://user-images.githubusercontent.com/3065339/37343861-8ef0c0d0-26a7-11e8-889d-80819e3648c7.png)

**Comment by camsaul** [comment]
Fixed by #7232

**Comment by salsakran** [comment]
Fixed in 0.28.4 which just went out. @valtlfelipe
